src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-02-studio-70.md

```diff
@@ -0,0 +1,49 @@
+---
+title: October 2, 2017
+description: This version polished autocompletion in automation scripting, and improves flow between Modeler and Designer.
+tree_item_index: 999999995
+review:
+    comment: ''
+    date: '2017-10-02'
+    status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v70'}}
+This version improves flow between Designer and Modeler, polishes autocompletion in automation scripting and brings bug fixes.
```

Comment:
_This version improves flow between Designer and Modeler as well as auto-completion in automation scripting and some bug fixes._